src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,81 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, ColonyClientV6, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeMotionLimits(
+      _,
+      { colonyAddress, userAddress, motionId, tokenDecimals },
+    ) {
+      try {
+        const colonyClient = (await colonyManager.getClient(
+          ClientType.ColonyClient,
+          colonyAddress,
+        )) as ColonyClientV6;
+        const votingReputationClient = await colonyClient.getExtensionClient(
+          Extension.VotingReputation,
+        );
+        const {
+          skillRep,
+          stakes,
+          domainId,
+        } = await votingReputationClient.getMotion(motionId);
+        const { skillId } = await colonyClient.getDomain(domainId);
+        const { reputationAmount } = await colonyClient.getReputation(
+          skillId,
+          userAddress,
+        );
+        // @NOTE There's no prettier compatible solution to this :(
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        // eslint-disable-next-line max-len
+        const userMinStakeFraction = await votingReputationClient.getUserMinStakeFraction();
+
+        const totalStaked = bigNumberify(stakes[1])
+          .div(bigNumberify(10).pow(tokenDecimals))
+          .toNumber();
+        const totalStakeAmount = skillRep
+          .mul(totalStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 2))
+          .toNumber();
+        const userMinStakeAmount = skillRep
+          .mul(totalStakeFraction)
+          .mul(userMinStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 3))
+          .toNumber();
+        const formattedReputationAmount = reputationAmount
+          .div(bigNumberify(10).pow(tokenDecimals))
+          .toNumber();
```

Comment:
Again, as I mentioned in the other PR review:
- **don't** convert values toNumber, convert them toString as it's the only way to handle big values
- **don't** convert values to ETH before export, leave them as WEI, and use `Numeral` inside the component to display them

Reply:
The `ReactSlider` needs `number` values to set a min, max, and limit. 

How could not use them and provide what the Slider needs? Providing percentages instead of the number themself?

For example, if minStake is 250, maxStake is 500, and the requiredStake is 1000, then:

The min of the Slider is 25%(250), the max is 50%(500), and the max is 100%(1000) ? 

And to get the amount the user input with the slider we use a a rule of 3

What do you think?

Reply:
What about if you have a low value? 0.00001 

JS won't handle that, much less pass it to the React slider.

I think it's fine for now, but you have edge cases here which you are not covering